dart/lib/src/protocol/sentry_event.dart

```diff
@@ -187,10 +202,12 @@ class SentryEvent {
         contexts: contexts ?? this.contexts,
         breadcrumbs: breadcrumbs ?? this.breadcrumbs,
         sdk: sdk ?? this.sdk,
+        request: request ?? this.request,
       );
 
   /// Serializes this event to JSON.
-  Map<String, dynamic> toJson({String origin}) {
+  Map<String, dynamic> toJson([String origin = '']) {
+    print('SentryEvent.toJson  $origin');
```

Comment:
if you think this is important to be logged, please use `options.logger` instead

Reply:
✅  removed